gcn/notices/icecube/LVK_nu_track_search.example.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/gcn/notices/icecube/LVK_nu_track_search.schema.json",
+  "type": "IceCube LVK Alert Nu Track Search",
+  "ref_ID": "MS230427r",
+  "alert_type": "initial",
```

Comment:
Apologies for confusion, later we had discussion over it. These fields can create confusion in data-base (in future) like alert_type of parent (neutrino) or LVK (follow-up). So it would be useful to use "reference" field for followUp core-schema, which will be directed to LVK notice. This alert_type and record_number should be used for parent instrument like IceCube here.